stdlib/pervasives.gr

```diff
@@ -112,7 +120,7 @@ export primitive (||): (Bool, Bool) -> Bool = "@or"
  *
  * @since v0.1.0
  */
-export (==)
+provide { (==) }
```

Comment:
All of these make me pretty sad. We should rework the place graindoc pulls from to be the original definition so we can use one provide to expose all from the file.

Can you open an issue?

Reply:
https://github.com/grain-lang/grain/issues/1626